test/stat_tracker_test.rb

```diff
@@ -1,12 +1,13 @@
 require './test/test_helper'
 require './lib/stat_tracker'
+require 'mocha/minitest'
 
 class StatTrackerTest < Minitest::Test
 
   def setup
-    game_path = './data/games.csv'
+    game_path = './data/games_fixture.csv'
```

Comment:
Thanks for doing this, I noticed it too when my test were taking forever to run.